app/src/main/java/com/github/se/cyrcle/ui/addParking/attributes/AttributesPicker.kt

```diff
@@ -131,6 +135,15 @@ fun AttributesPicker(
             price = 0.0,
             uid = parkingViewModel.getNewUid())
     parkingViewModel.addParking(parking)
+
+    // Credit coins to user
```

Comment:
Same as for the Add review, this should be a function of the viewmodel so that it only takes one line in the UI